PhysicsTools/NanoAODTools/test/exampleAnalysis.py

```diff
@@ -42,7 +42,7 @@ def analyze(self, event):
 
 
 preselection = "Jet_pt[0] > 250"
-files = ["file:/eos/cms/store/group/cat/datasets/NANOAODSIM/RunIISummer20UL18NanoAODv9-106X_upgrade2018_realistic_v16_L1v1-v2/DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/7B930101-EB91-4F4E-9B90-0861460DBD94.root"]
+files = ["root://cms-xrd-global.cern.ch//store/group/cat/datasets/NANOAODSIM/RunIISummer20UL18NanoAODv9-106X_upgrade2018_realistic_v16_L1v1-v2/DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/7B930101-EB91-4F4E-9B90-0861460DBD94.root"]
```

Comment:
dear #smuzaffarm 
The first solution is not viable because this is not a cmsRun script (it runs in plain py3).
I will pull a change to root://eoscms.cern.ch//eos/cms/store/user/cmsbuild tomorrow.

Reply:
One option would be to use `edmFileUtil` to translate the LFN of `/store/group/cat/datasets/...` to the PFN according to the same catalog mechanism as `cmsRun` does.

Reply:
@makortel that is a good suggestion and I implemented it directly into the postprocessor ([here](https://github.com/cms-sw/cmssw/pull/42323/commits/347bd36aa1d19eae3d69cddcc19c9f0edadeb850#diff-b977c03549b92fa7b68e5f4be0f624e76631c04b9261cfc6d7a95eaaa1bad25fR159)).
However, for this file `edmFileUtil ` translates to:
`root://eoscms.cern.ch//eos/cms/store/group/cat/datasets/...` 
while @smuzaffar suggested to take: 
`root://eoscms.cern.ch//eos/cms/store/user/cmsbuild/...` 
so I set the latter explicitly in the test scripts.

Reply:
ah I did not know `edmFileUtil` can convert LFN to PFN. @namapane , I am fine if you use `root://eoscms.cern.ch//eos/cms/store/user/cmsbuild` directly or make use of `edmFileUtil -d` option. Note that for PR tests and IBs `edmFileUtil` will return the ibeos cache path as we explicitly set `SITECONFIG_PATH="/cvmfs/cms-ib.cern.ch/SITECONF/local"` during these tests

```
lxplus> export SITECONFIG_PATH="/cvmfs/cms-ib.cern.ch/SITECONF/local"                                                                                lxplus> edmFileUtil -d  /store/group/cat/datasets/NANOAODSIM/RunIISummer20UL18NanoAODv9-106X_upgrade2018_realistic_v16_L1v1-v2/DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/7B930101-EB91-4F4E-9B90-0861460DBD94.root

root://eoscms.cern.ch//eos/cms/store/user/cmsbuild/store/group/cat/datasets/NANOAODSIM/RunIISummer20UL18NanoAODv9-106X_upgrade2018_realistic_v16_L1v1-v2/DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/7B930101-EB91-4F4E-9B90-0861460DBD94.root
```

Reply:
OK good to know... I'd keep the full PFN anyhow if that's not a problem,first to avoid a futher commit. Also as we want to keep some basic support for standalone usage, and users in standalone mode taking these scripts as  an example would run into a missing edmFileUtil otherwise.